### PR TITLE
New Tool: WP-CLI

### LINKED ISSUE #187: Add support for WP-CLI as a tool.
**Is your feature request related to a problem? Please describe.**
I need **[WP-CLI](https://wp-cli.org)** to be in the `bin` for [this](https://github.com/kidunot89/wp-graphql/blob/feature/github-workflows/.github/workflows/schema-linter.yml) workflow 

**Describe the solution you'd like**
Add the **wp-cli.phar** as **wp** in the `bin`.

**Describe alternatives you've considered**
I made [this](https://github.com/kidunot89/wp-cli-github-action) action ~ It didn't work :cry: 

**Additional context**
I've already devised a solution. See #188 .

### PR DESCRIPTION
**Pull request by kidunot89**
---
name: 🎉 Adds WP-CLI as a tool.
about: You found a bug, want to improve something or add a new feature
labels: enhancement

---

## A Pull Request should be associated with an Issue.

> We wish to have discussions in Issues. A single issue may be targeted by multiple PRs.
> If you're offering a new feature or fixing anything, we'd like to know beforehand in Issues,
> and potentially we'll be able to point development in a particular direction.

Related issue: 
Closes #187 

### Description

This PR adds support WP-CLI as a **tool**. The windows setup is lacking because I don't know PowerShell. Sorry :man_shrugging:.

> In case this PR introduced TypeScript/JavaScript code changes:

- [x] I have written test cases for the changes in this pull request
- [x] I have run `npm run format` before the commit.
- [x] I have run `npm run lint` before the commit.
- [x] I have run `npm run release` before the commit.
- [x] `npm test` returns with no unit test errors.

### CODE REVIEW

File: src/tools.ts [hunk 1]
```diff
@@ -308,6 +308,26 @@ export async function getSymfonyUri(
   }
 }
 
+/**
+ * Function to get the Deployer url
```
Comment (shivammathur) [inline src/tools.ts]: Change Deployer to wp-cli

File: src/scripts/darwin.sh [hunk 2]
```diff
@@ -109,6 +109,8 @@ add_tool() {
         sudo sed -i '' 's/exit(9)/exit(0)/' "$tool_path"
         tr -d '\r' < "$tool_path" | sudo tee "$tool_path.tmp" >/dev/null 2>&1 && sudo mv "$tool_path.tmp" "$tool_path"
         sudo chmod a+x "$tool_path"
+      elif [ "$tool" = "wp-cli" ]; then
+        sudo mv "$tool_path" /usr/local/bin/wp
```
Comment (shivammathur) [inline src/scripts/darwin.sh]: change `mv` to `cp`, so that both `wp` and `wp-cli` work.

File: src/scripts/linux.sh [hunk 3]
```diff
@@ -165,6 +165,8 @@ add_tool() {
       add_extension curl "$apt_install php$version-curl" extension >/dev/null 2>&1
       add_extension mbstring "$apt_install php$version-mbstring" extension >/dev/null 2>&1
       add_extension xml "$apt_install php$version-xml" extension >/dev/null 2>&1
+    elif [ "$tool" = "wp-cli" ]; then
+      sudo mv "$tool_path" /usr/local/bin/wp
```
Comment (shivammathur) [inline src/scripts/linux.sh]: change `mv` to `cp`, so that both `wp` and `wp-cli` work

Comment (kidunot89) [reply src/scripts/linux.sh]: Done

File: src/scripts/darwin.sh (hunk 2 above)
Comment (kidunot89) [reply src/scripts/darwin.sh]: Done

File: src/tools.ts (hunk 1 above)
Comment (kidunot89) [reply src/tools.ts]: Done

**Pull request merged**